Submissions/sggts04/Task_2/task2.py

```diff
@@ -0,0 +1,12 @@
+import os
+import shutil
+
+os.chdir(os.path.join(os.getcwd(), "My_Files"))
```

Comment:
Try to refer the folder which we gave instead of copying My_Files in this folder

Reply:
You can use for example os.chdir('../../../ur_folder')

Reply:
Fixed now